its/src/test/javaSuite/java.test.ts

```diff
@@ -14,6 +14,7 @@ import * as vscode from 'vscode';
 // import * as myExtension from '../extension';
 
 const sampleFolderLocation = '../../../samples/';
+const sampleJavaFolderLocation = '../../../samples/sample-java-maven-multi-module/';
```

Comment:
Why duplicating almost the same constant than the previous one? You could keep this one and use it as well line 33

Reply:
Fixed.